glauth/tasks/main.yaml

```diff
@@ -0,0 +1,28 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+---
+- name: Install package
+  ansible.builtin.package:
+    name: glauth
+    state: present
+
+- name: Create user
```

Comment:
I think we should move this to the debian package

Reply:
Yup, already done (https://github.com/ASSEKURANSA/glauth/commit/7d958bbdc3e8c012c3e77e538830120b08ce65c4).

This, then, should be a no-op, though I would like to leave it in, because of the `group: glauth` in the next task. Thoughts?

Reply:
ok, fair enough